README.md

```diff
@@ -1,49 +1,75 @@
+- In _app.js_: The block at the very top, commented with _Important_
+  ```JavaScript
+  const mongodb_port = process.env.DBPORT || '27017';
+  const db = require('monk')(`127.0.0.1:${mongodb_port}/omm-2223`); // connect to database omm-2021
+  console.log(`Connected to MongoDB at port ${mongodb_port}`)
+  ```
+- In _package.json_: The _scripts_ block
+  ```JSON
+  "scripts": {
+    "startdev": "node ./bin/www",
+    "start": "SET DBPORT=65535 && node ./bin/www"
+  },
+  ```
 
-For development we recommend to work with the two project (_node-backend_ and _react-frontend_) individually. However if you prefer, you can use the mememuc-launcher during development too, with the following two commands:
+The other two folders __must not be changed__!
+* `./mememuc-launcher` contains configuration files for installing dependencies and launching the application you implement.
+* `./mongoserver` cotains a local in-memory database server independent of any existing local installation.
+  This database is not persistent and will reset with each restart. It is meant for testing your submission with a consistent data state, independent of the computer on which it runs.
+  You _can_ add files to the `./mongoserver/data` subdirectory.
 
-`cd mememuc-launcher`
-Navigates your commandline into the mememuc launcher project
 
-`npm run installall`
-Installs the dependencies of all (sub)projects
+## How To Use
 
-`npm run startdev`
-The backend project will connect to a local MongoDB instance (assuming it is running on your localhost, at the default port 27017).
+### During Development
 
+During development we recommend to ruin the two project (`./node-backend` and `./react-frontend`) individually. However if you prefer, you can use the NodeJS scripts from `./mememuc-launcher` during development too, with the following commands:
 
-### Submission
+```
+cd mememuc-launcher
+```
+navigates your commandline into the mememuc launcher project
 
-To take a look on your submission, we will launch the following two commands:
+```
+npm run installall
+```
+installs the dependencies of all (sub)projects
 
-`cd mememuc-launcher`
-Navigates your commandline into the mememuc launcher project
+```
+npm run startdev
+```
+starts the backend project. It will connect to a local MongoDB instance (assuming any is running on your local machine) at the default port `27017`.
 
-`npm run installall`
-Installs the dependencies of all (sub)projects
+### How To Prepare Your Submission
 
-`npm start`
-The backend project will connect to a non-persistent in-memory MongoDB instance.
+- Export the MongoDB database state that you want us to user for evaluating your submission from your local MongoDB as json files. You can use, e.g., the `mongoexport` command:
+`mongoexport --uri="mongodb://localhost:27017/omm-ws2223" --collection=users --out=omm-ws2223.json`
```

Comment:
Using the default mongo port makes more sense here, I reckon, since they are meant to export from their local database, right?